client/index.php

```diff
@@ -190,6 +189,13 @@
 try {
     $jiraWebhook->run();
 } catch (\Exception $e) {
+    // For convenience in local development show errors on screen directly

```

Comment:
Oh, I like this! thanks @alebedev80 . I'll check it out, see if we can log both in the log file and in stdout